data-raw/watershed/regional_approximation/Regional_Approximation.Rmd

```diff
@@ -0,0 +1,168 @@
+---
+title: "Regional Approximation"
+author: "[Sadie Gill](mailto:sgill@flowwest.com), [Mark Tompkins](mailto:mtompkins@flowwest.com), [Erin Cain](mailto:ecain@flowwest.com)"
+date: "Aug 8, 2019"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
+knitr::opts_knit$set(root.dir = rprojroot::find_rstudio_root_file())
+library(tidyverse)
+library(readxl)
+library(cvpiaHabitat)
+library(purrr)
+library(lubridate)
+```
+
+
+## Upper-mid Sacramento Region 
+
+### Instream Spawning and Rearing Habitat
+
+**Details:**    
+A mean regional weighted usable area (WUA) was developed for watersheds without instream spawning and rearing habitat modeling. 
+
+For unmodeled tributaries of the Upper Sacramento River, the rearing WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, Clear Creek, and Cow Creek. The spawning WUA values were estimated using the mean WUA at each flow from Battle Creek, Butte Creek, and Clear Creek.
+
+**Spawning Not Modeled:** 
+
+* Antelope Creek
```

Comment:
plz double check that the watershed's listed are still correct as you have added some new modeling